### PR TITLE
Change message for API version mismatch in Kibana

### PR DESCRIPTION
**Pull request by Tostti**
Solves #4299 
# Important information
On PR https://github.com/wazuh/wazuh-kibana-app/pull/4103, a method was changed to request the API endpoint. Only for testing purposes of this particular issue, it is possible to replace line 50 of `server/lib/api-interceptor.ts` with the following: `method: !!authContext ? 'POST' : 'GET',` to avoid getting an error due to a not allowed method and trigger the desired error.

# Changes made:

- Modified the message displayed when there is a version mismatch between the Wazuh API and the Wazuh App, to make it easier to understand.
- Removed the \<a\> tags on the message. 
- Added `PLUGIN_APP_NAME` constant to make the message match the same structure as in Wazuh Dashboard, for easier modifications in the future. 
- Added `addTagsToUrl` function on `health-check-container.tsx` and modified the displayed error message to use that function. 

## `addTagsToUrl`
This function receives a string and returns it with the next modifications:

- If the string contains a structure `<something> guide: <URL>`, it will modify that part to return `<something> guide` wrapped in \<a\> tags with the href pointing to the URL. 
- If the string contains a URL without `guide:` before it, it will wrap the URL in \<a\> tags.
- If the string doesn't contain any URL, it will return it without modifications.

## Before the changes: 
image

## After the changes:
image

# Tests
**Scenario**: have an environment with different versions of Wazuh API and Wazuh app, in which the major and minor are not the same.
**When** the user opens the Wazuh plugin
**Then** the health-check confirmation should show an error on `Check Wazuh API Version`
**And** the log message should contain a URL to the Elasticsearch troubleshooting section of Wazuh Documentation.
**And** the log message should not contain any \<a\> tags or HTML structure
**And** the message shown at the bottom should contain a clickable link to the Elasticsearch troubleshooting section of Wazuh Documentation.

**Scenario**: have an environment with different versions of Wazuh API and Wazuh app, in which the major and minor are not the same.
**When** the user forces a health check in debug mode
**Then** the health-check confirmation should show an error on `Check Wazuh API Version`
**And** the log message should contain a URL to the Elasticsearch troubleshooting section of Wazuh Documentation.
**And** the log message should not contain any \<a\> tags or HTML structure
**And** the message shown at the bottom should contain a clickable link to the Elasticsearch troubleshooting section of Wazuh Documentation.

### CODE REVIEW

**Comment by Tostti** [comment]
# Changes done

- Modified the message to be clearer and removed the <a> tags on it. 
- Modified the URL shown in the message to the troubleshooting URL. 
- Added a function on `health-check.container.tsx` to add the corresponding tags to URLs in error messages, thus there are shown with a working link. 

Before the changes:
image

After the changes:
image

File: public/components/health-check/services/check-setup.service.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const checkSetupService = appInfo => async (checkLogger: CheckLogger) =>
         api.groups.version !== appSplit[0] ||
         api.groups.minor !== appSplit[1]
       ) {
-        checkLogger.error(`Wazuh API and Wazuh App version mismatch. API version: ${apiVersion}. App version: ${setupData.data.data['app-version']}. At least, major and minor should match. Check more info about upgrading Wazuh App <a target='_blank' href='${webDocumentationLink(PLUGIN_PLATFORM_WAZUH_DOCUMENTATION_URL_PATH_UPGRADE_PLATFORM)}'>here</a>.`);
+        checkLogger.error(`Wazuh API and ${PLUGIN_APP_NAME} version mismatch. API version: ${apiVersion}. App version: ${setupData.data.data['app-version']}. Check more info about this error on our troubleshooting guide: ${webDocumentationLink(PLUGIN_PLATFORM_WAZUH_DOCUMENTATION_URL_PATH_TROUBLESHOOTING)}.`);
```
Comment (AlexRuiz7) [inline public/components/health-check/services/check-setup.service.ts:47]: I think that: `Read more about this error in our troubleshooting guide:` sounds better

Comment (Tostti) [reply public/components/health-check/services/check-setup.service.ts]: Talked in daily, modified in 6eebd86221204bfe87af16d7ebaa8beae2b403ba

**Review by Mayons95 — APPROVED** [review]
LGTM

**Pull request merged**